src/main/java/duke/Launcher.java

```diff
@@ -0,0 +1,29 @@
+package duke;
+
+import duke.program.DukeCli;
+import duke.program.DukeGui;
+import javafx.application.Application;
+
+public class Launcher {
+    /**
+     * The entry point to the Duke CLI program.
+     *
+     * @param filepath The location of the storage file.
+     */
+    public static void launchCli(String filepath) {
```

Comment:
I think since we are using RCS, it is preferred to remove dead code from the codebase.